joomla/joomla-cms
# [4.0] Correct minimum version check for MariaDB on installation

Pull Request for Issue [https://github.com/joomla/joomla-cms/pull/25285#issuecomment-504428587](https://github.com/joomla/joomla-cms/pull/25285#issuecomment-504428587) .

### Summary of Changes

- Add new UI text `INSTL_DATABASE_INVALID_MARIADB_VERSION` to installation language files.
- When check for minimum DB version fails with mysqli or mysql driver, check if MariaDB and if so, use the right UI text for the exception.

### Testing Instructions

1. Try to install current 4.0-dev on a MariaDB of a version which does not fulfill the minimum reguirement 10.0.0. Result: See section "Actual result" below.
2. Apply the changes of this PR and try again. Result: See section "Expected result" below.

### Expected result

Error message like `You need MariaDB 10.0 or higher to continue the installation. Your version is: 5.5.60-MariaDB`.

### Actual result

Error message like `You need MySQL 10.0 or higher to continue the installation. Your version is: 5.5.60-MariaDB`.

### Documentation Changes Required

None.
